kernel/driver.cc

```diff
@@ -150,6 +150,9 @@ const char *prompt()
 
 #else /* EMSCRIPTEN */
 
+char Yosys::yosys_path[PATH_MAX];
+char *Yosys::yosys_argv0;
+
```

Comment:
I set that here to look alike the rest of the source, but maybe you will have a better place for them.